addons/popochiu/engine/objects/graphic_interface/templates/sierra/sierra_gi.gd

```diff
@@ -0,0 +1,117 @@
+extends PopochiuGraphicInterface
+
+
+# ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ GODOT ░░░░
+func _ready() -> void:
+	super()
+	
+	Cursor.replace_frames($Cursor)
+	Cursor.show_cursor()
+	
+	$Cursor.hide()
+	
+	$Menu.hide()
+	
+	E.current_command = SierraCommands.Commands.WALK
+	
+	%SierraSettingsPopup.option_selected.connect(_on_settings_option_selected)
+
+
+func _input(event: InputEvent) -> void:
+	if D.current_dialog: return
+	
+	if event is InputEventMouseMotion:
+		if get_global_mouse_position().y < 16.0:
+			# Show the top menu
+			if not I.active:
+				Cursor.show_cursor("gui")
+			
+			$Menu.show()
+		elif get_global_mouse_position().y > $Menu.size.y and $Menu.visible:
+			# Hide the top menu
+			if not I.active:
+				Cursor.show_cursor(E.get_current_command_name(true))
+			
+			$Menu.hide()
+	elif event is InputEventMouseButton and event.is_pressed():
+		match (event as InputEventMouseButton).button_index:
+			MOUSE_BUTTON_LEFT:
+				if not $Menu.visible and not E.hovered\
+				 and E.current_command != SierraCommands.Commands.WALK:
+					get_viewport().set_input_as_handled()
```

Comment:
Maybe you can add a comment explaining that you are inhibiting the default popochiu walking action.

Reply:
Done.